src/philips_airctrl/discovery.py

```
@@ -0,0 +1,154 @@
+                    )
+
+                    logger.info("Found Philips air purifier at %s: %s", ip, device_info.model)
+                    return device_info
+
+                finally:
+                    await client.shutdown()
+
+            except TimeoutError:
+                logger.debug("Timeout connecting to %s", ip)
+                return None
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)